MAUI/Scheduler/migration.md

```diff
@@ -2301,9 +2592,6 @@ this.Content = scheduler;
 
 ## Upcoming Features
 
-* Resource view support.
-* Days count support in timeslot views.
-* Month view navigation direction support.
 * Command support for Tapped, DoubleTapped, LongPressed and ViewChanged events.
```

Comment:
* Command support for Tapped, DoubleTapped, LongPressed, and ViewChanged events.